src/lib.rs

```diff
@@ -1,6 +1,7 @@
 //! Create and parses JWT (JSON Web Tokens)
 //!
```

Comment:
More docs could be added here to briefly describe all the features of the library (encoding, decoding, verification, reserved claim verification, etc)